pkg/config/config_template.yaml

```diff
@@ -3905,3 +3905,16 @@ api_key:
   #
   # log_file: <TRAY_LOG_FILE_PATH>
 {{end -}}
+{{- if (eq .OS "windows")}}
+
+#####################################################
+## Datadog Agent Windows Crash Detection module
+#####################################################
+
+# windows_crash_detection:
+  ## @param enable - boolean - optional - default: false
+  ## Enables the system probe module which supports the Windows crash detection check.
```

Comment:
Could we add a bit more info here around this feature or a link to the public docs?

Reply:
I feel that the product is still being defined 🤷‍♂️

Reply:
The product is not still being defined.  This is the result of a PM-defined feature, with a product brief, and the output has been defined & approved by PM.